capture/listener.go

```diff
@@ -640,7 +638,7 @@ func (t *Listener) readPcapFile() {
 				continue
 			}
 
-			dataOffset := (data[12] & 0xF0) >> 4
+			dataOffset := data[12] >> 4
```

Comment:
Copy\paste from https://github.com/buger/goreplay/pull/768/files#diff-cb5774656d51d7a2d177f84ee666c512R523.
Is it possible to move to separate func?

Reply:
moving it to another function will cause an un-necessary increase in CPU cycles which will decrease performance!

Reply:
did you try it? so now it ok, but can you please create some example to show increase CPU and decrease performance?